es/es-repository-hdfs/src/main/java/org/elasticsearch/repositories/hdfs/HdfsRepository.java

```diff
@@ -51,7 +51,7 @@
 
 public final class HdfsRepository extends BlobStoreRepository {
 
-    private static final Logger LOGGER = LogManager.getLogger(HdfsRepository.class);
```

Comment:
should the name still stay capitalized?

Reply:
no need really - in all other repos, it's in lowercase, I prefer to make it consistent with all others

Reply:
static vars should always be uppercase